### PR TITLE
Add basic generator

### PR DESCRIPTION
**Pull request by mallardduck**
Hey there @owenvoke - this is just a PR I'm opening to show how simply https://github.com/blade-ui-kit/blade-icons/pull/144 could help maintain icon sets. Please feel free to test it yourself as I didn't put much time into verifying what normalization steps you are doing currently.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,12 +9,19 @@
         "illuminate/support": "^8.0"
     },
     "require-dev": {
+        "ext-dom": "*",
```
Comment (owenvoke) [inline composer.json:12]: I guess this is no longer needed?

Comment (mallardduck) [reply composer.json:12]: Correct - just overlooked that after I looked at how your action worked to match that. 😄

Comment (mallardduck) [reply composer.json:12]: I'll make sure to address removing this when I remove the `"repositories"` key after the PR is merged in at blade-icons.

**Comment by owenvoke** [comment]
Thank you! I'm so glad this feature was merged. I'm going to review this again on Sunday evening/Monday night when I'm around again, and hopefully merge. Thank you @mallardduck for your work. :+1:

**Comment by mallardduck** [comment]
Super happy the generator is merged in and released now! I've updated the PR here to use the stable release so that will be read when you test.

**Review by owenvoke — APPROVED** [review]

**Pull request merged**